src/main/java/com/minecolonies/entity/ai/citizen/lumberjack/EntityAIWorkLumberjack.java

```diff
@@ -507,17 +501,17 @@ private void tryGettingUnstuckFromLeaves()
      */
     private BlockPos findNearLeaves()
     {
-        int playerX =   worker.getPosition().getX();
-        int playerY =   worker.getPosition().getY() + 1;
-        int playerZ =   worker.getPosition().getZ();
-        int radius  = 3;
+        int playerX = worker.getPosition().getX();
+        int playerY = worker.getPosition().getY() + 1;
+        int playerZ = worker.getPosition().getZ();
+        int radius  = LEAVES_RADIUS;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'radius' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)